src/stdune/fpath.mli

```diff
@@ -0,0 +1,5 @@
+type mkdir_p =
```

Comment:
Could you add a comment explaining what this module is about? The name is not obvious.